src/lib/mongoBackend/collectionOperations.h

```diff
@@ -0,0 +1,47 @@
+#ifndef COLLECTIONOPERATIONS_H
```

Comment:
The IDE chose COLLECTIONOPERATIONS_H unnoticelly. I will change to SRC_LIB_MONGOBACKEND_COLLECTIONOPERATIONS_H_